public/locales/en/common.json

```diff
@@ -175,5 +175,31 @@
   "planetCashTC": "By clicking \"Create PlanetCash Account\", I agree to the <0>Terms and Conditions</0> of Plant-for-the-Planet Platform, and PlanetCash.",
   "planetCashIUnderstand": "I understand that PlanetCash donation/payments are non-refundable, and unallocated payments will be auto allocated to projects on the Platform after six months of Payment.",
   "createPlanetCashAccount": "Create PlanetCash Account",
-  "myContributionIsPartOfThat": "And my contribution is a part of that."
+  "myContributionIsPartOfThat": "And my contribution is a part of that.",
+  "notFound": "Resource not found",
+  "unauthorized": "Not Authorized",
+  "validationError": "Validation Error",
```

Comment:
It's little bit weird these strings are used in `@planet-sdk/common`, but have to be provided in the translations of the project using the SDK. 

How can one know if new translations have to be added or existing ones have to change or can be deleted if the SDK changes?

Even more complicated seems to be some of these translations are not even defined by the SDK, but by a string of the response from the Backend. So if the Backend changes these translations have to be adapted as well. 

So we have a dependency here from the SDK and the Backend. We should at least document this somewhere to remember it.

Reply:
The documentation for Error Handling is already present on [notion](https://www.notion.so/plantfortheplanet/Error-Handling-in-React-Apps-9cb00e89731a48d79be53c83345c10d0).

I have updated the doc to include all [translation keys](https://www.notion.so/plantfortheplanet/Error-Handling-in-React-Apps-9cb00e89731a48d79be53c83345c10d0#57d6a2d201ec4e5d8830958cb3f30ff6).